htslib/vcf.h

```diff
@@ -756,29 +781,56 @@ set to one of BCF_ERR* codes and must be checked before calling bcf_write().
 
     /**
      *  bcf_get_info_*() - get INFO values, integers or floats
-     *  @hdr:       BCF header
-     *  @line:      BCF record
-     *  @tag:       INFO tag to retrieve
-     *  @dst:       *dst is pointer to a memory location, can point to NULL
-     *  @ndst:      pointer to the size of allocated memory
+     *  @param hdr:    BCF header
+     *  @param line:   BCF record
+     *  @param tag:    INFO tag to retrieve
+     *  @param dst:    *dst is pointer to a memory location, can point to NULL
+     *  @param ndst:   pointer to the size of allocated memory
+     *  @return  0 on success
```

Comment:
There is discrepancy between the return code for success and the paragraph below, which states:
`Returns negative value on error or the number of written values (including missing values) on success.`